scripts-available/CDB_Overviews.sql

```diff
@@ -204,7 +231,7 @@ AS $$
         -- This is the typical ERROR: stats for "mytable" do not exist
         WHEN internal_error THEN
           -- Get stats and execute again
-          EXECUTE format('ANALYZE %1$I', reloid);
+          EXECUTE format('ANALYZE %1$s', reloid);
```

Comment:
I hoped you knew! (you placed that code there). Is it supposed to generate the stats, the lack of which could have provoked an error?